opentelemetry-sdk/benches/noop_metrics.rs

```diff
@@ -0,0 +1,85 @@
+use criterion::{criterion_group, criterion_main, Criterion};
+use opentelemetry::{
```

Comment:
since we don't need sdk for these tests, could we keep them as part of opentelemetry project itself?

Reply:
yes good point. Will move it there.